.changeset/nice-humans-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+You can now view gift card details page if you have only gift card permissions and customer details page if you have only customer permissions.
```

Comment:
question: What was previous behavior? Could we also add that to the changeset?

Reply:
Done ✅